src/app_service/service/community/service.nim

```diff
@@ -1695,10 +1695,11 @@ QtObject:
     except Exception as e:
       error "Error banning user from community", msg = e.msg
 
-  proc setCommunityMuted*(self: Service, communityId: string, muted: bool) =
+  proc setCommunityMuted*(self: Service, communityId: string, mutedType: int) =
     try:
-      discard status_go.setCommunityMuted(communityId, muted)
-
+      discard status_go.setCommunityMuted(communityId, mutedType)
+      
+      let muted = if (MutedType(mutedType) == MutedType.Unmuted): false else: true
```

Comment:
If we don't relay on actual value from status-go, it's better to handle an error:
```
      let response = status_go.setCommunityMuted(communityId, mutedType)
      if not response.error.isNil:
        error "error muting the community", msg = response.error.message
        return
```